R/util.R

```diff
@@ -119,17 +119,14 @@ get_nested_costs <- function(raw_costs) {
 #'
 #' @description Convert daily GVA values to annual GDP values.
 #'
-#' @param x A string giving a country name from among `daedalus::country_names`.
+#' @param x A string giving a country name from among `daedalus::country_names`
+#' or an ISO2 code from among `daedalus::country_codes_iso2c` or an ISO3 code
+#' from among `daedalus::country_codes_iso3c`.
 #'
 #' @return A single number value for the annual GDP of a country in terms of
 #' million dollars. Values are in 2018 terms.
 #' @keyword internal
 get_annual_gdp <- function(x) {
-  stopifnot(
-    "`x` must be a string giving a country name from `daedalus::country_names" =
-      is.character(x) && x %in% daedalus::country_names
-  )
-
```

Comment:
Removed this check as the call to daedalus_country below will throw an appropriate error if provided code or name is not a match,